.gitignore

```diff
@@ -0,0 +1,104 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
```

Comment:
I guess we do not need so much stuff here, but I leave it to your choice.

Reply:
done